doc/manual.lyx

```diff
@@ -2246,11 +2246,11 @@ Labels
 \end_layout
 
 \begin_layout Standard
-The ddg2labels tool may be used to generate a .csv file of the input ∆∆Gs
+The pdb2lavels tool may be used to generate a .csv file of the input ∆∆Gs
```

Comment:
There's a little typo - it should be pdb2labels rather than pdb2lavels. Same for the other occurrences